index.js

```diff
@@ -13,6 +13,9 @@ var xtend = require('xtend')
 
 module.exports = Choo
 
+Choo.html = require('./html')
+Choo.raw = require('./html/raw')
```

Comment:
This will add choo/html to yo-yoified browserify bundles too, increasing bundle size by a few KB… I think people should probably use `nanohtml` directly instead for UMD, or we could have a separate UMD entry file that does something like
```js
module.exports = require('./')
module.exports.html = require('./html')
module.exports.raw = require('./html/raw')
```

Reply:
Good point. Reverting.